website/content/en/events/2024-cloud-native-sustainability-week.md

```diff
@@ -29,7 +29,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | **NR** | **Location** | **Date** | **Event Link** | **Name** |
 |---|---|---|---|---|
 | 1 | Amsterdam, Netherlands | 10th October | [link](https://community.cncf.io/e/mwszqh/) | Julie Chenadec, Alessandro Vozza, Pavel Chunyayev, Matteo Bianchi
-| 2 | Guatemala | TBD | TBD | Sergio Méndez
+| 2 | Guatemala | Oct. 26 | [Link](https://community.cncf.io/e/m8chk2/) | Diana Menendez, Sergio Méndez
```

Comment:
@sergioarmgpl thanks for the PR! Can you please also update the same file for different languages? Here's an example from the earlier PR: https://github.com/cncf/tag-env-sustainability/pull/505/files

Reply:
Sure I will do it